ui/apps/notifications/src/components/app-routes.tsx

```diff
@@ -34,6 +34,7 @@ const AppRoutes = (props: RootComponentProps) => {
               description={t(
                 'Get the latest updates about what’s going on with your world. You can personalize your notifications and get only what you want to see!',
               )}
+              image={isLoggedIn ? '/images/notificationapp-welcome-min.webp' : null}
```

Comment:
better just pass the image and move the logic to display when logged it inside the component

Reply:
Thanks for noticing it Vali :)